### ISSUE TITLE
[bug] customField LDAP syncronization is inconsistent (partly broken)

### ISSUE THREAD

**Issue by ankar84** [RocketChat/Rocket.Chat#17752]
### Description:

Our Rocket Chat users all imported from **LDAP (Microsoft Active Directory).**
We also use **customFields** in Rocket Chat, that also imported from LDAP.
Here is how we describe **customFields** in **Admin UI - Account - Registration**
````
{
	"telephoneNumber": {
		"type": "text",
		"required": true,
		"minLength": 0,
		"maxLength": 12,
		"public": true
	},
    "email": {
		"type": "text",
		"required": true,
		"minLength": 0,
		"maxLength": 50,
		"public": true
	},
	"Title": {
		"type": "text",
		"required": true,
		"minLength": 0,
		"maxLength": 50,
		"public": true
	},
	"Department": {
		"type": "text",
		"required": true,
		"minLength": 0,
		"maxLength": 50,
		"public": true
	},
	"l": {
		"type": "text",
		"required": true,
		"minLength": 0,
		"maxLength": 20,
		"public": true
	}
}
````

And here is a **User Data Field Map**configured in **Admin UI - LDAP - Sync/Import**
````
{ "displayName": "name","mail": "email" ,"telephoneNumber":"customFields.telephoneNumber","title":"customFields.Title","department":"customFields.Department","l":"customFields.l"}
````
When we first added **customFields** to our Rocket Chat users all worked fine - all user had all **customFields** filled and up to date.
But now we are on Rocket Chat version 3.1.1 and now **customFields** for many users not fully filled.
I think that's because some information was changed in origin (LDAP) and on synchronization customField that was changed in LDAP was not updated in Rocket Chat, but was deleted. 
We made some experiment on that problem.
First we cleared all **customFields** for problem user - and after Background Synchronization - all **customFields** was filled with current information, which is great.
But then we changed one field (for example **telephoneNumber**) in Rocket Chat user info to reproduce data changes in LDAP and after that **telephoneNumber** was cleared in Rocket Chat user info. Then we changed **Title** in RC user info and after **LDAP Background Synchronization** **Title** was **updated correctly** and **telephoneNumber come back again**, but **l (city)**
 and **Department** was deleted (cleared).
So - we don't understand how does it works now - randomly update/delete random customField.
That should be fixed.
@pierre-lehnen-rc please take a look at that issue

### Steps to reproduce:

1. Configure **customFields** from LDAP
2. Update one of **customFields** in LDAP
3. That **customFields** is deleted in Rocket Chat user information

### Expected behavior:

All **customFields** are updated when that data is updated in LDAP (origin for that data)

### Actual behavior:

Update of **customFields** is not consistent right now.

### Server Setup Information:

- Version of Rocket.Chat Server: 3.1.1
- Operating System: CentOS7
- Deployment Method: docker
- Number of Running Instances: 25
- DB Replicaset Oplog: Enabled
- NodeJS Version: 12.16.1
- MongoDB Version: 4.0.17

### Client Setup Information

- Desktop App or Browser Version: 2,17,7 and Chome 83
- Operating System: Windows 10

**Comment by ankar84** [comment]
I found that my issue is duplicate of https://github.com/RocketChat/Rocket.Chat/issues/15374

**Comment by ankar84** [comment]
I think it could be already fixed in my 3.6.3 version.
I will check it in more detail and come back.

**Comment by ankar84** [comment]
Looks like it fixed in 3.6.3 
@sampaiodiego @pierre-lehnen-rc
